src/app/src/test/kotlin/org/imperial/mrc/hint/database/VersionRepositoryTests.kt

```diff
@@ -272,6 +290,63 @@ class VersionRepositoryTests
         assertThat(originalVersionDetails).isEqualToComparingFieldByFieldRecursively(clonedVersionDetails)
     }
 
+    @Test
+    fun `can clone version and note to new project`()
+    {
+        val uid = setupUser()
+        val uid2 = setupUser("another.user@email.com")
+
+        val originalProject = projectRepo.saveNewProject(uid, "p1")
+        val originalVersionId = "v1"
+        sut.saveVersion(originalVersionId, originalProject)
+        sut.saveVersionState(originalVersionId, originalProject, uid, "{'something': 1}")
+        setUpHashAndVersionFile("survey_hash", "survey_file", originalVersionId, "survey", false)
+
+        val clonedProject = projectRepo.saveNewProject(uid2, "p1", note = "test note")
+        val clonedVersionId = "v2"
+        sut.cloneVersion(originalVersionId, clonedVersionId, clonedProject)
```

Comment:
Unless you add the `note` parameter here then this whole test may be redundant - it doesn't appear to actually be asserting anything related to notes